specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -266,7 +266,7 @@ export class BusinessRuleManager<SCHEMA extends AnySchema> {
 
     const invalidResponse: BusinessRuleResult<SCHEMA> = {
       valid: false,
-      /* eslint-disable-next-line */
+      /* eslint-disable-next-line array-func/prefer-includes */
       reason: fieldInfo.some((field) => field === undefined)
```

Comment:
hmm, why was this ESLint rule disabled? seems like a valid suggestion

`fieldInfo.includes(undefined)`

Reply:
ups, stale comment. this is what happens when you review one commit at a time